composer.json

```diff
@@ -12,7 +12,12 @@
   },
   "autoload": {
     "psr-4": {
-      "Google\\CloudFunctions\\": "src"
+      "Google\\CloudFunctions\\": "src/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "Google\\CloudFunctions\\Tests\\": "tests/"
```

Comment:
Why do we need this when we're not autoloading any classes in this namespace, and phpunit doesn't require it? 

From [composer.org](https://getcomposer.org/doc/04-schema.md#autoload-dev):
> Classes needed to run the test suite should not be included in the main autoload rules to avoid polluting the autoloader in production and when other people use your package as a dependency.

I agree with this, but since we're not loading any classes it makes more sense to me to leave this out until we do.

Reply:
Not sure this conforms with most standards, but up to you

Reply:
I am just not sure what the purpose of the lines are if they are not autoloading any classes. But If I'm missing something please let me know!

Reply:
I think what composer.org says is that if you autoload dependencies they should absolutely not be in the `autoload` section if they are test-related, thus the introduction of this `autoload-dev` section.

Reply:
We don't autoload test dependencies, which is why I do not think `autoload-dev` is needed.